### PR TITLE
3850-Migrate-Hiedra-to-Spec2

### LINKED ISSUE #3850: Migrate Hiedra to Spec2
I'll do a PR

### PR DESCRIPTION
**Pull request by jecisc**
Migrate Hiedra to Spec 2Â + clean code

Fixes #3850

### CODE REVIEW

**Comment by jecisc** [comment]
Test failure seems not related.

**Comment by Ducasse** [comment]
Cyril 
Julien ported hiedra to Spec2. Is it this one?

**Comment by jecisc** [comment]
No. I was not aware of it. 

I know that Julien started to post Epicea to Spec 2 and I will wait the end of his holidays to migrate Epicea with him.

File: src/Hiedra-Tests/HiRulerBuilderTest.class.st [hunk 1]
```diff
@@ -10,6 +10,11 @@ Class {
 	#category : #'Hiedra-Tests-Model'
 }
 
+{ #category : #'initialize - event' }
+HiRulerBuilderTest class >> initialize [
+	DebugShowingRulerInWindow := false
+]
```
Comment (tinchodias) [inline src/Hiedra-Tests/HiRulerBuilderTest.class.st:16]: thanks!

**Pull request merged**